internal/auth0/organization/resource_connection.go

```diff
@@ -60,13 +76,18 @@ func createOrganizationConnection(ctx context.Context, data *schema.ResourceData
 	api := meta.(*config.Config).GetAPI()
 
 	organizationID := data.Get("organization_id").(string)
-
 	connectionID := data.Get("connection_id").(string)
 	assignMembershipOnLogin := data.Get("assign_membership_on_login").(bool)
 
+	// We need to keep these null if they don't appear in the schema
```

Comment:
These will never be `null` if we are setting default values above. So either we remove the default values above, or keep the default values but use `data.Get` to fetch them, just as above with the `assignMemberShipOnLogin` var.